docker-compose-postgresql.yml

```diff
@@ -96,12 +96,12 @@ services:
       - FINERACT_DEFAULT_TENANTDB_DESCRIPTION=Default Demo Tenant
 
   # Frontend service
-  community-app:
-    image: openmf/community-app:latest
-    container_name: mifos-ui
-    restart: always
-    ports:
-      - 9090:80
-    depends_on:
-      fineract-server:
-        condition: service_healthy
+  #community-app:
```

Comment:
I guess you want to remove this or uncomment?

Reply:
I'd remove it here... and leave the separate docker-compose-community-app.yml as a convenience for anyone who used it... then no one can complain.